ui/app/index.html

```diff
@@ -47,7 +47,7 @@
                <a ui-sref="home.streams.tabs.definitions">Streams</a>
             </li>
             <li ng-class="{ active: $state.includes('home.jobs') }">
```

Comment:
Would be good to replace the variable name `job(s)` to `task(s)` for consistency.